js/models/roomcollection.js

```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```

Comment:
The lastPing is already returned, that's what the join does ;) 

The return of `/apps/spreed/api/room` is:

```
[
   {
      "id":"26",
      "name":"test",
      "type":"1",
      "userId":"admin",
      "roomId":"26",
      "lastPing":"1476883919",
      "displayName":"test",
      "validRoom":true,
      "count":0
   },
   {
      "id":"27",
      "name":"test1",
      "type":"1",
      "userId":"admin",
      "roomId":"27",
      "lastPing":"1476883922",
      "displayName":"Dick",
      "validRoom":true,
      "count":0
   }
]
```